app/views_redesigned/creatives/forms/_standard_form.html.erb

```diff
@@ -100,38 +100,40 @@
-                    data-style="btn-sm btn-subtle-primary"
-                    data-target="creative-form.inputTemplateTheme"
-                    data-action="change->creative-form#setTemplateTheme">
-              <option value="default|light">Default (light)</option>
-              <option value="default|dark">Default (dark)</option>
-              <option value="centered|light">Centered (light)</option>
-              <option value="centered|dark">Centered (dark)</option>
-              <option value="horizontal|light">Horizontal (light)</option>
-              <option value="horizontal|dark">Horizontal (dark)</option>
-              <option value="vertical|light">Vertical (light)</option>
-              <option value="vertical|dark">Vertical (dark)</option>
-              <option value="image-centered|light">Image Centered (light)</option>
-              <option value="image-centered|dark">Image Centered (dark)</option>
-              <option value="image-only|light">Image Only (light)</option>
-              <option value="image-only|dark">Image Only (dark)</option>
-              <option value="square|light">Square (light)</option>
-              <option value="square|dark">Square (dark)</option>
-            </select>
-          </li>
-        </ul>
+      <div class="col-12">
+        <h6 class="text-center">Preview</h6>
       </div>
+      <%
+        # <div class="col-sm-8 col-md-6 text-sm-right" hidden>
+        #   <ul class="list-inline mb-0">
+        #     <li class="list-inline-item">
+        #       <select class="js-select selectpicker"
+        #               data-style="btn-sm btn-subtle-primary"
+        #               data-target="creative-form.inputTemplateTheme"
+        #               data-action="change->creative-form#setTemplateTheme">
+        #         <option value="default|light">Default (light)</option>
+        #         <option value="default|dark">Default (dark)</option>
+        #         <option value="centered|light">Centered (light)</option>
+        #         <option value="centered|dark">Centered (dark)</option>
+        #         <option value="horizontal|light">Horizontal (light)</option>
+        #         <option value="horizontal|dark">Horizontal (dark)</option>
+        #         <option value="vertical|light">Vertical (light)</option>
+        #         <option value="vertical|dark">Vertical (dark)</option>
+        #         <option value="image-centered|light">Image Centered (light)</option>
+        #         <option value="image-centered|dark">Image Centered (dark)</option>
+        #         <option value="image-only|light">Image Only (light)</option>
+        #         <option value="image-only|dark">Image Only (dark)</option>
+        #         <option value="square|light">Square (light)</option>
+        #         <option value="square|dark">Square (dark)</option>
+        #       </select>
+        #     </li>
+        #   </ul>
+        # </div>
+      %>
```

Comment:
commented this out since its not being used but it wasn't clear at first. I assume there was a reason this was left in the code...?